adapters/rest-adapter-generator/src/test/java/uk/gov/justice/services/adapters/rest/generator/DefaultGenerator_ResourceMethodBodyTest.java

```diff
@@ -11,40 +40,13 @@
 import org.mockito.ArgumentCaptor;
 import org.mockito.Mock;
 import org.mockito.runners.MockitoJUnitRunner;
+
 import uk.gov.justice.raml.core.GeneratorConfig;
 import uk.gov.justice.services.adapter.rest.RestProcessor;
-import uk.gov.justice.services.adapters.test.utils.JavaCompilerUtil;
+import uk.gov.justice.services.adapters.test.utils.compiler.JavaCompilerUtil;
 import uk.gov.justice.services.core.dispatcher.Dispatcher;
 import uk.gov.justice.services.messaging.Envelope;
 
-import javax.json.Json;
-import javax.json.JsonObject;
-import javax.ws.rs.core.HttpHeaders;
-import javax.ws.rs.core.Response;
-import java.lang.reflect.Field;
-import java.lang.reflect.Method;
-import java.nio.file.Path;
-import java.nio.file.Paths;
-import java.util.Arrays;
-import java.util.List;
-import java.util.Map;
-import java.util.Optional;
-import java.util.function.Consumer;
-import java.util.stream.Collectors;
-
-import static org.hamcrest.Matchers.is;
-import static org.junit.Assert.assertThat;
-import static org.junit.Assert.assertTrue;
-import static org.mockito.Matchers.any;
-import static org.mockito.Matchers.eq;
-import static org.mockito.Mockito.verify;
-import static org.mockito.Mockito.when;
-import static org.raml.model.ActionType.POST;
-import static uk.gov.justice.services.adapters.test.utils.ActionBuilder.action;
-import static uk.gov.justice.services.adapters.test.utils.RamlBuilder.restRamlWithDefaults;
-import static uk.gov.justice.services.adapters.test.utils.ResourceBuilder.resource;
-import static uk.gov.justice.services.messaging.DefaultEnvelope.envelopeFrom;
-
 @RunWith(MockitoJUnitRunner.class)
 public class DefaultGenerator_ResourceMethodBodyTest {
 
```

Comment:
probably a result of Eclipse's organise imports (Ctrl+Shift+O) to remove unused imports